### ISSUE TITLE
Mobile station with -- in name not accepted

### ISSUE THREAD

**Issue by mase76** [renyuneyun/Easer#438]
Hi,
at my home place I have a mobile station with -- in its name, which is not accepted in easer. I don't think, that the German Telekom cares about this. Maybe you can accept such names in Easer.

**Comment by renyuneyun** [comment]
That's interesting. Did you find the station through the search/listen UI button (on the cellular station event) on Easer?

When it was implemented, Android document said the cellular stations will only have two parts of numbers connected together. Maybe something changed now?

**Comment by mase76** [comment]
Yes, I found it via the button in Easer. Maybe a fault of my provider. But not fixed.

**Comment by renyuneyun** [comment]
Thanks for the update. Do you have any examples? I may try to fix that if the examples seem reasonably compatible with existing schema. Otherwise I'll read the documents again to see if there is any updates.

**Comment by mase76** [comment]
I have to wait, until I see this station again. Stay tuned...

**Comment by mase76** [comment]
Sorry for the long wait. This cell id is still there:
65966400--1
I changed some numbers, but see the two --
